dev/drivers/scripts/global_ens/plots/jevs_global_ens_atmos_gefs_grid2grid_past31days_plots.sh

```diff
@@ -1,13 +1,12 @@
-#PBS -N jevs_global_ens_grid2grid_past31days_plots
+#PBS -N jevs_global_ens_atmos_gefs_grid2grid_past31days_plots
```

Comment:
Question here:  why are we adding the "gefs" into the filename and PBS output option?

Reply:
It matches the script name.

Reply:
I am confused here. These plots feature more than one model series. Why would GEFS be needed in the filename in that case?

Reply:
The ex-scripts are named this way. https://github.com/NOAA-EMC/EVS/tree/develop/scripts/global_ens/plots

Reply:
Then my follow-up is whether the ex-scripts should be named that way if they are handling multiple models (not just gefs)?Or is it just trivial?

Reply:
I have no idea. I didn't name them.

Reply:
Alright, I think now understand the naming convention for both the global_ens plots drivers and ex-scripts.